.TOOLVERSIONS

```diff
@@ -1,5 +1,5 @@
-android-ndk;r10e;070be287539e3e7706f8dabfb6bf9879
-android-sdk-build-tools;28.0.1;
+android-ndk;r17c;a4b6b8281e7d101efd994d31e64af746
+android-sdk-build-tools;28.0.3;
 android-sdk-platform;android-27;
```

Comment:
@PombeirP we also need to use android-28, can we change this as well? Or this config doesn't have effect on docker image?

Reply:
OK, I'll push a new image with that change too. In your #status-core message you mentioned

> we need to upgrade android-sdk-build-tools to 28.0.3 and android-sdk-platform to android-27

that's why I didn't change it

Reply:
oh I see, that was a mistake, sorry